doc/compiling_dependencies.rst

```diff
@@ -0,0 +1,73 @@
+Compiling dependencies
+========================
+
+1. Setting up compilers
+
+E.g. on SupermucNG:
+
+.. code-block:: bash
+
+    module load intel mpi-intel
+    export FC=mpif90
+    export CXX=mpiCC
+    export CC=mpicc
+
+Alternatively for gcc compilers on SupermucNG:
+
+.. code-block:: bash
+
+    export FC=mpigcc
+    export CXX=mpigxx
+    export CC=mpifc
+
+2. Asagi
+ 
+See https://seissol.readthedocs.io/en/latest/asagi.html.
+
+3. yaml-cpp
+
+On SupermucNG and with intel compilers:
+
+.. code-block:: bash
+
+    module load yaml-cpp/0.6.3-intel19
+
+    
+Alternatively to build from source:
+
+.. code-block:: bash
+
+    git clone git@github.com:jbeder/yaml-cpp
+    cd yaml-cpp
```

Comment:
This should probably include a 
`git checkout yaml-cpp-0.6.3`
Not sure whether version 0.7 actually works; and in this way we can ensure that it won't break in the future.